src/partial.lenses.js

```diff
@@ -1278,3 +1278,26 @@ export const json = /*#__PURE__*/(process.env.NODE_ENV === "production" ? I.id :
   return isoU(expect(I.isString, text => JSON.parse(text, reviver)),
               expect(I.isDefined, value => JSON.stringify(value, replacer, space)))
 })
+
+// Interop
+
+const isArray = x =>
+  !(x instanceof Object) || Array.isArray(x)
```

Comment:
Hmm... I would rename this to something slightly different (this is not quite the same as e.g. `Array.isArray` and I'd like to avoid possible confusion with that) and move this upwards in the file into the utilities section.  For historical reasons (to avoid module overhead) almost all of the PL code is in a single file, but after 12.0.0 I should really refactor the code into smaller files and take advantage of Rollup's ability to eliminate module overhead.

Reply:
The two most difficult things in computer science are cache invalidation, naming and off by one errors.

Any suggestion for a better name?

Reply:
I don't see a section marked 'utilities', so it may make sense for you to rename this and move it where you think it best fits. I'll be happy to rename and move it with more detailed instructions to make sure I get it right. Either way is fine for me.